.buildkite/pipelines/on_merge_unsupported_ftrs.yml

```diff
@@ -85,3 +85,5 @@ steps:
       automatic:
         - exit_status: '*'
           limit: 1
+        - exit_status: 10
+          limit: 0
```

Comment:
@elastic/kibana-operations if there is reserved status code for similar cases (intentional failure without a retry), I'm happy to replace `10` for consistency.